tgui/packages/tgui/interfaces/Keybinds/Footer.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Garash2k (https://github.com/garash2k)
+ * @license MIT
+ */
+import { Button } from 'tgui-core/components';
+
+import { useBackend } from '../../backend';
+import { KeybindsData } from './types';
+
+export const Footer = () => {
+  const { act, data } = useBackend<KeybindsData>();
+  const { hasChanges, resetting } = data;
+
+  return (
+    <>
+      <Button
+        onClick={() => act('confirm')}
+        color={!hasChanges ? null : 'good'}
+        icon="save"
+      >
+        Confirm
+      </Button>
+      <Button
+        onClick={() => act('reset')}
+        color={!resetting ? null : 'bad'}
+        icon="trash"
+      >
+        {!resetting ? 'Reset All' : 'Confirm?'}
```

Comment:
Having both a `Confirm` button (the one before this) and this one (`Confirm?` after you've pressed `Reset All` once) is a bit odd. I also don't see a way to get out of the resetting state short of saving your keybinds.

Reply:
I was suggested have the same behavior as the Character Setup "Reset All" button, for which you can't get out of the resetting state either. Looking it up now though, it's a Button.Confirm so I'll use that, and I'll use "Confirm Reset All?" to disambiguate.